appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy

```diff
@@ -16,13 +16,19 @@ import nu.yona.server.test.User
 
 class UserPhotoTest extends AbstractAppServiceIntegrationTest
 {
-	def 'User uploads user photo'()
+	static final EXAMPLE_PNG_DATA_BASE64 = "iVBORw0KGgoAAAANSUhEUgAAABIAAAAVCAYAAABLy77vAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAlSURBVDhPY/j98dl/auBRgwjjUYMI41GDCONRgwjjUYMI4Wf/AVx5oubiBf17AAAAAElFTkSuQmCC"
+
+	def setupSpec()
+	{
+		appService.yonaServer.restClient.encoder.putAt("multipart/form-data", new MethodClosure(this, 'encodeMultipartEntity'))
```

Comment:
Add a comment along these lines:
// Register MIME type encoder to call our method encodeMultipartEntity

Reply:
Will add